phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
+		    $get_task_index = (int)$this->iohandler->get_input('task_index', 0);
+		    // Validation check
+		    if($get_task_index > 0 && $get_task_index < $iterator->count()){
+                $get_task_index--;
+                $iterator->seek($get_task_index);
+                // Setting the menu point as active
+                $this->iohandler->set_active_stage_menu($iterator->current()->get_navigation_stage_path());
```

Comment:
Although the same applies as below, you already had that information client side, why would you want to send it back from the server.